packages/backend/src/server.test.ts

```diff
@@ -263,6 +263,70 @@ describe('api', () => {
       expect(response.id).toEqual(expect.any(String))
     })
 
+    test('update submission state - approved', async () => {
+      const submissionResult = await post(
+        `http://localhost:${port}/api/submissions`,
+        { ...absaSubmission, invitationId: invitation.id },
+        cookie,
+      )
+      const status = submissionResult.status
+      const submissionResponse = await submissionResult.json()
+      expect(status).toEqual(201)
+      expect(submissionResponse.id).toEqual(expect.any(String))
+      const approvalResult = await put(
+        `http://localhost:3000/api/submissions/${submissionResponse.id}`,
+        { state: 'approved' },
+        cookie,
+      )
+      const approvalResponse = await approvalResult.json()
+      expect(approvalResponse.submission.state).toEqual('approved')
+      expect(approvalResponse.entity).toBeDefined()
+
+      // Check that the entity is added to the registry
+      const registry = await fetchRegistry()
+      expect(registry.entities).toContainEqual(approvalResponse.entity)
+    })
+
+    test('update submission state - rejected', async () => {
+      const submissionResult = await post(
+        `http://localhost:${port}/api/submissions`,
+        { ...absaSubmission, invitationId: invitation.id },
+        cookie,
+      )
+      const status = submissionResult.status
+      const response = await submissionResult.json()
+      expect(status).toEqual(201)
+      expect(response.id).toEqual(expect.any(String))
+      const rejectionResult = await put(
+        `http://localhost:3000/api/submissions/${response.id}`,
+        { state: 'rejected' },
+        cookie,
+      )
+      const updatedSubmission = await rejectionResult.json()
+      expect(updatedSubmission.state).toEqual('rejected')
+    })
+
+    test('update submission state - unknown', async () => {
```

Comment:
Does it mean that the state is gonna be changed to `uknown`? :) The name suggest that according to other tests names and their behaviour, if I understand them correctly.